soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -11,3 +11,15 @@ config SOC_IT8XXX2
 	select ATOMIC_OPERATIONS_C
 
 endchoice
+
+choice
+	prompt "Select clock source for PLL reference clock"
+	default SOC_IT8XXX2_INT_32K
```

Comment:
"default" can be dropped here. The default is always the first for choice symbols

Reply:
Done